### ISSUE TITLE
AIL integration (via leak-hunter)

### ISSUE THREAD

**Issue by adulau** [Felix83000/Watcher#1]
Watcher is a cool idea and full of potential. Another interesting connection would be the integration with [AIL](https://github.com/ail-project/ail-framework) to feed matching leaks from AIL into Watcher.

**Comment by Felix83000** [comment]
Yes, for sure, it will be very useful for Watcher, we will highly considerate this idea! 👍
